internal/releases/repo_github.go

```diff
@@ -41,41 +41,91 @@ func (r *githubRepository) Process() error {
 	ctx := context.Background()
 	opts := &github.ListOptions{PerPage: 3}
 
+	// Check if the repository is archived
+	repoObject, _, err := client.Repositories.Get(ctx, r.org, r.info.Name)
+	if err != nil {
+		return fmt.Errorf(
+			"error while checking archived status for repo: %s/%s/%s: %v", r.org, r.team, r.info.Name, err,
+		)
+	}
+	r.info.IsArchived = repoObject.GetArchived()
```

Comment:
If the repo is archived, and we're not going to display it, surely it'd be better to bail out here and just set the releases/commits/newcommits to the empty values? Saves a bunch of extra API calls and processing.

Reply:
True!

Not sure if it makes sense to have `IsArchived` at all since it's not used by the release page at the moment; I think having the field itself doesn't cost us much and it's more *future-proof* (maybe at some point we want to show archived repos with a switch?), so I'd leave it there and just exit from the function